src/Microsoft.AspNet.Razor/Parser/CSharpCodeParser.Statements.cs

```diff
@@ -126,31 +126,40 @@ private void UsingDeclaration()
             // Set block type to directive
             Context.CurrentBlock.Type = BlockType.Directive;
 
-            // Parse a type name
-            Assert(CSharpSymbolType.Identifier);
```

Comment:
Great to see this `Assert()` go away. Using `Debug.Assert()` to inform the user of an error parsing their Razor file was more than a bit odd.